src/Sdk/ApiClient.ts

```diff
@@ -105,4 +90,8 @@ export default class ApiClient {
             query,
         });
     }
+
+    private buildEndpointUrl(endpointUri: string): string {
+        return `${stripSlashes(this.baseUrl)}/${stripSlashes(endpointUri)}`;
```

Comment:
I'm doing this because I don't want to make any assumtions of the `baseUrl` since it can be a value from outside the module.
I think it's easier and better DX if we sanitize the url to how we want them instead of explaining what the value should be and leaving options for a small mistake like a trailing slash.